drivers/media/i2c/ov5693.c

```diff
@@ -1765,6 +1761,9 @@ static int ov5693_probe(struct i2c_client *client)
 	if (ret)
 		ov5693_remove(client);
```

Comment:
Not really part of this PR, but shouldn't there be a return or something here and on the `if (ret)` above? Otherwise you're overwriting the `ret` value below and continue to probe even though `ov5693_init_controls()` or `media_entity_pads_init()` may have failed.

Reply:
Yes, agree. There's a lot of stuff like that throughout tbh - I think we've had so many people hacking at it that it's gotten a bit wild. I'm going to go through function by function and iron out problems like that when I get time...but after I get the exposure fixed.